tests/llm/test_llm.py

```diff
@@ -0,0 +1,285 @@
+import pytest
+from pydantic import BaseModel
+
+from chatsky.llm._langchain_imports import HumanMessage, AIMessage, langchain_available
+from chatsky.llm.llm_api import LLM_API
+from chatsky.responses.llm import LLMResponse
+from chatsky.llm.utils import message_to_langchain, attachment_to_content, context_to_history
+from chatsky.llm.filters import IsImportant, FromTheModel
+from chatsky.llm.methods import Contains, LogProb
+from chatsky.core.message import Message, Image
+from chatsky.core.context import Context
+from chatsky.core.script import Node
+from chatsky.core.node_label import AbsoluteNodeLabel
+from chatsky.llm._langchain_imports import LLMResult, HumanMessage
+from langchain_core.outputs.chat_generation import ChatGeneration
```

Comment:
In all of the tests: extra dependencies may not be available.
Import from `langchain` only after it has been confirmed (via `lanchain_available`) that it is present:
```python
if not langchain_available:
    pytest.skip(allow_module_level=True, reason="Langchain not available.")
```